messages/rest.md

```diff
@@ -33,10 +33,33 @@ Make an authenticated HTTP request to Salesforce REST API and print the response
 
   <%= config.bin %> <%= command.id %> 'sobjects/account/<Account ID>' --body "{\"BillingCity\": \"San Francisco\"}" --method PATCH
 
+- You can store every flag option as a parameter in a json file, with the following schema:
+  {
+  body?: string;
+  header?: string[];
+  url?: string;
+  method?: string;
+  }
+
+  looking at the example above, we could store all of this information in the file, and change the command to
+
+  <%= config.bin %> <%= command.id %> --file myFile.json
+
+  where myFile.json contains
+  {
+  "url": "sobjects/Account/<Account ID>",
+  "method": "PATCH",
+  "body" : {"BillingCity": "Boise"}
+  }
+
 # flags.method.summary
 
 HTTP method for the request.
 
+# flags.file.summary
+
+A json file to store values for header/body/method/url - this is the same format as a Postman Collection Format
```

Comment:
@WillieRuemmele @VivekMChawla I couldn't find a way to export a postman request in the collection format.
If it's not possible then we might not need to follow their format?

context:
by not following their format we leave this open to us adding support for parameterized params later:
https://salesforce.quip.com/5t19A9rykzI3#SdNADAvRyDL